dojo/engagement/views.py

```diff
@@ -641,7 +641,7 @@ def import_scan_results(request, eid=None, pid=None):
 
             if not error:
                 return HttpResponseRedirect(
-                    reverse('view_test', args=(test.id, )))
+                    reverse('product_open_findings', args=(pid, )))
```

Comment:
I do not believe this is where imports should return to. In the case of a product having existing findings before an import, the user would have no idea which findings were imported in the most recent import than those that already existed.

Reply:
The problem that this page he redirects is unfinished, causing always error, what idea would you have for that? I could help with this task.

Reply:
Can you elaborate on what unfinished means here? It has been set to redirect to the view test page for years without issue.

Reply:
@pabloalexsandroalmeida83 what error are you seeing? The view_test page is like 100 years old, so should be working :-) Maybe fix the error in the parser and then see if the view_test page works.